src/Garnek.java

```diff
@@ -0,0 +1,16 @@
+public class Garnek {
+
+        int high;
+        int width;
+        String color;
+
+       public Garnek(){
+       }
```

Comment:
The number of empty lines between elements should be constant so please add an empty line between these two constructors and remove reduntant empty lines below second constructor